test/fixtures/silent_payments.json

```diff
@@ -0,0 +1,823 @@
+[
+  {
+    "comment": "Simple send: two inputs",
```

Comment:
what's the difference between this test and the next one (`Simple send: two inputs, regtest`)?